src/IB/IBFEMethod.cpp

```diff
@@ -1882,7 +2423,14 @@ IBFEMethod::computeStressNormalization(PetscVector<double>& Phi_vec,
                                                            PK1_grad_var_data[k],
                                                            data_time,
                                                            d_PK1_stress_fcn_data[part][k].ctx);
-                        Phi += n * ((PP * FF_trans) * n) / J;
+                        if (d_phi_current_config)
+                        {
+                            Phi += n * ((PP * FF_trans) * n) / J;
```

Comment:
I would suggest adding a variable instead of this `if` statement:
```
                        double volume_scale = (d_phi_current_config ? 1.0 / J : 1.0);
                        Phi += n * ((PP * FF_trans) * n) * volume_scale;
```
(Same comment below.)